### DISCUSSION TITLE
[BP#30] Préférer les glyphs aux images

### DISCUSSION THREAD

**Discussion by florinesueur** [cnumr/best-practices#225] [Bonnes pratiques] [answered]
Ici il n'est fait que mention de l'utilisation des glyphes d'une font système.

Il est aussi possible de créer une icons-font ou d'ajouter à son fichier de font utilisé sur le site des images vectorielles.

Pour un site, j'avais 3 illustrations et 5 pictogrammes monochromes totalisant au total 233ko en svg et 149 ko en png. On peut en déduire que le spreedsheet est légèrement supérieur en termes de poids. L'icons-font en woff2 créée avec tous les éléments pèse 19ko avec un petit bonus SEO et accessibilité avec les ligatures comme google icons :  
```
<span class="icons-font">
lock_clock
</span>
```

Ceci est en relation avec la discussion sur les spritesheets #223 et la [BP20](https://github.com/cnumr/best-practices/blob/main/chapters/BP_020_fr.md).

**Comment by ACTLEM** [comment, marked as answer]
J'ai créé l'issue pour améliorer la description (voire exemple): https://github.com/cnumr/best-practices/issues/258
